dcm4che-conf/dcm4che-conf-core/src/test/java/org/dcm4che3/conf/core/tests/adapters/DefaultConfigTypeAdaptersTest.java

```diff
@@ -0,0 +1,117 @@
+package org.dcm4che3.conf.core.tests.adapters;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.dcm4che3.conf.core.adapters.DefaultConfigTypeAdapters.EnumTypeAdapter;
+import org.dcm4che3.conf.core.adapters.ReflectiveAdapter.PropertySchema;
+import org.dcm4che3.conf.core.api.ConfigurableClass;
+import org.dcm4che3.conf.core.api.ConfigurableProperty;
+import org.dcm4che3.conf.core.api.ConfigurationException;
+import org.dcm4che3.conf.core.api.internal.ConfigProperty;
+import org.dcm4che3.conf.core.api.internal.ConfigReflection;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+class DefaultConfigTypeAdaptersTest {
+
+	@Nested
+	class EnumTypeAdapterTests {
+	
+		/**
+	     * System Under Test (SUT).
+	     */
+	    private EnumTypeAdapter adapter = new EnumTypeAdapter();
+	    
+		@Test
+		void getSchema_ReturnsCorrectlyPopulatedMap_GivenValidEnumConfigProperty() {
+
+			List<ConfigProperty> configurableFields = ConfigReflection.getAllConfigurableFields(TestEnumClass.class);
+
+			final Map<String, Object> actualSchema = adapter.getSchema(configurableFields.get(0), null);
+			
+	        Map<String, Object> expectedSchema = new HashMap<>();

```

Comment:
Spacing looks off in this file.

Reply:
will fix it, I had some issues with tabs vs spaces in IDE